javascript-data-model/order-history.js

```diff
@@ -1,30 +1,30 @@
 var orderHistory = [
   {
-    item: 'JavaScript for impatient programmers',
-    author: 'Rauschmayer, Dr. Axel',
-    price: '$31.55',
-    orderPlaced: 'August 4, 2020',
-    orderDelivered: 'Aug 8, 2020'
+    item: ['JavaScript for impatient programmers'],
+    author: ['Rauschmayer, Dr. Axel'],
+    price: ['$31.55'],
+    orderPlaced: ['August 4, 2020'],
+    orderDelivered: ['Aug 8, 2020']
   },
   {
-    item: 'The Timeless Way of Building',
-    author: 'Alexander, Christopher',
-    price: '$41.33',
-    orderPlaced: 'July 19, 2020',
-    orderDelivered: 'Jul 20, 2020'
+    item: ['The Timeless Way of Building'],
+    author: ['Alexander, Christopher'],
+    price: ['$41.33'],
+    orderPlaced: ['July 19, 2020'],
+    orderDelivered: ['Jul 20, 2020']
   },
   {
-    item: 'Gamecube Controller Adapter Super Smash Bros Switch Gamecube Adapter for WI U, PC Support Turbo and Vibration Features No Driver and No Lag Gamecube Adapter',
-    price: '$15.98',
-    orderPlaced: 'July 4, 2020',
-    orderDelivered: 'Jul 7, 2020'
+    item: ['Gamecube Controller Adapter Super Smash Bros Switch Gamecube Adapter for WI U, PC Support Turbo and Vibration Features No Driver and No Lag Gamecube Adapter'],
+    price: ['$15.98'],
+    orderPlaced: ['July 4, 2020'],
+    orderDelivered: ['Jul 7, 2020']
```

Comment:
This will totally work, but rather than having a bunch of arrays, I suggest having one array with an object for each item in the order.